### ISSUE TITLE
Provide more context for the notification status screen

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#601]
(This is about this 'jobs' page - bulk send, rather than the notifications page - anything you've sent in the last 7 days)

## Problem statement
[It wasn't immediately clear for one participant what the send report screen was communicating. However, report table communicated errors correctly (systematically) to participants.
](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwEUFgBd0KLqH7oo/recAcnInIAzMqlGM3?blocks=hide)

[Numbers on the send report don't provide enough context for people using a screen reader](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwBYyaQxOttZ0IAw/reccOy0fnXUUsDrvr?blocks=hide)

[Not immediately clear what the status report page is of
](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwBYyaQxOttZ0IAw/recS40o66HtQtGTp0?blocks=hide)

[Send report filters don't seem clickable](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwBYyaQxOttZ0IAw/recity730NIFpUwaI?blocks=hide)

## Hypothesis
A bit more context for people with no/low vision people could help them orient themselves on this page faster.

**Comment by YedidaZalik** [comment]
@yaelberger-commits could we add this to bounce rate v1.5?

**Comment by amazingphilippe** [comment]
Scoped some work out of this research story, I don't think we can tackle everything at once. The cards are linked and we should update this card once #1262 is complete. 

[Ideation for future work can be found in Miro](https://miro.com/app/board/uXjVO6TPyIs=/?moveToWidget=3458764557654836884&cot=14), awaiting further story refinement/mapping

**Comment by adriannelee** [comment]
Going to wait on bringing this card to story refinement so that we have a little bit more info on the notification status screen, through bounce rate testing.

**Comment by amazingphilippe** [comment]
[Some insights from Fable testing.](https://cds-snc.dovetailapp.com/insights/3ljy9QYKrsKVwnc7beMCBS) We have enough information to confirm the labeling of the number+legend is unclear to more users. We could investigate a CSS fix to provide the optimal experience to both visual and screen reader users.
